mmdet/models/dense_heads/rpn_head.py

```diff
@@ -22,14 +23,28 @@ class RPNHead(AnchorHead):
     def __init__(self,
                  in_channels,
                  init_cfg=dict(type='Normal', layer='Conv2d', std=0.01),
+                 num_convs=1,
                  **kwargs):
+        self.num_convs = num_convs
         super(RPNHead, self).__init__(
             1, in_channels, init_cfg=init_cfg, **kwargs)
 
     def _init_layers(self):
         """Initialize layers of the head."""
-        self.rpn_conv = nn.Conv2d(
-            self.in_channels, self.feat_channels, 3, padding=1)
+        if self.num_convs > 1:
+            rpn_convs = []
+            for i in range(self.num_convs):
+                rpn_convs.append(
+                    ConvModule(
+                        self.in_channels,
+                        self.feat_channels,
+                        3,
+                        padding=1,
+                        inplace=False))
+            self.rpn_conv = nn.Sequential(*rpn_convs)
```

Comment:
Need to pay attention to the relu operation in the forward code.

Reply:
relu is handled in ConvModule